client/tests/e2e/specs/adminPlanning.js

```diff
@@ -31,7 +31,7 @@ describe('Planning tests', () => {
     cy.adminDisconnection()
   })
 
-  it('Assigns a candidate and changes the inspector', () => {
+  it('Assigns a candidate, send bordereaux and changes the inspector of booked place', () => {
```

Comment:
Est-on obligé de faire ces 3 actions dans le même test ?

Reply:
Oui, j'ai jugé cela plus judicieux car si je crée un nouveau `it()` je vais devoir répéter l’opération d'affection et de vérification qui est faite en amont de l'envoi des bordereaux. De plus le test initial  effectuai déjà les test des bordereaux sauf que le l'information n'était pas renseigner dans la description du `it()`